pkgs/tools/admin/aws-sso-cli/default.nix

```diff
@@ -1,25 +1,33 @@
-{ buildGoModule, fetchFromGitHub, lib }:
-  buildGoModule rec {
-    pname = "aws-sso-cli";
-    version = "1.9.2";
+{ buildGoModule
+, fetchFromGitHub
+, lib
+, makeWrapper
+, xdg-utils
+}:
+buildGoModule rec {
+  pname = "aws-sso-cli";
+  version = "1.9.2";
 
-    src = fetchFromGitHub {
-      owner = "synfinatic";
-      repo = pname;
-      rev = "v${version}";
-      sha256 = "9/dZfRmFAyE5NEMmuiVsRvwgqQrTNhXkTR9N0d3zgfk=";
-    };
-    vendorSha256 = "BlSCLvlrKiubMtfFSZ5ppMmL2ZhJcBXxJfeRgMADYB4=";
+  src = fetchFromGitHub {
+    owner = "synfinatic";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "9/dZfRmFAyE5NEMmuiVsRvwgqQrTNhXkTR9N0d3zgfk=";
+  };
+  vendorSha256 = "BlSCLvlrKiubMtfFSZ5ppMmL2ZhJcBXxJfeRgMADYB4=";
 
-    postInstall = ''
-      mv $out/bin/cmd $out/bin/aws-sso
-    '';
+  nativeBuildInputs = [ makeWrapper ];
 
-    meta = with lib; {
-      homepage = "https://github.com/synfinatic/aws-sso-cli";
-      description = "AWS SSO CLI is a secure replacement for using the aws configure sso wizard";
-      license = licenses.gpl3Plus;
-      maintainers = with maintainers; [ devusb ];
-      mainProgram = "aws-sso";
-    };
+  postInstall = ''
+    makeWrapper $out/bin/cmd $out/bin/aws-sso \
```

Comment:
now bin/cmd isn't moved to bin/aws-sso

do `mv` first and then `wrapProgram ...aws-sso \ ...`

Reply:
Could you explain your concern in more details? I mean the wrapper with the proper name is there and it executes the binary setting the required PATH variable, why does the binary matters? `makeWrapper` renames the binary anyway, doesn't it? 🤔

Reply:
makeWrapper doesn't rename the binary instead it creates a shell script that points to the binary

Reply:
When done with makeWrapper it looks like you still end up with a `cmd` binary in your PATH in addition to `aws-sso`. Looks like if you do `mv` from `cmd` to `aws-sso` first and then use `wrapProgram` instead of `makeWrapper` you will get the result you're looking for.